doc-content/enterprise-only/openshift/hacep-server-proc.adoc

```diff
@@ -5,6 +5,8 @@ The high-availability (HA) CEP server runs on the {OPENSHIFT} environment. It in
 
 You must prepare the source, build it, and then deploy it on {OPENSHIFT}. 
 
+Alternatively, you can use a different process to deploy the HA CEP server where you can update the KJAR service at any time. For instructions about this process, see <<hacep-server-maven-proc>>.
```

Comment:
Alternatively, use a different process to deploy the HA CEP server to update the KJAR service at any time. For more information, see <<hacep-server-maven-proc>>.